galaxy_ng/app/settings.py

```diff
@@ -1,3 +1,18 @@
+MIDDLEWARE = [
+    'django_prometheus.middleware.PrometheusBeforeMiddleware',
+    # BEGIN: Pulp standard middleware
+    'django.middleware.security.SecurityMiddleware',
+    'whitenoise.middleware.WhiteNoiseMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    'django.middleware.csrf.CsrfViewMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    # END: Pulp standard middleware
+    'django_prometheus.middleware.PrometheusAfterMiddleware',
+]
+
```

Comment:
Would like to figure out if there is a way to alter the existing MIDDLEWARE instead of replacing it, but I didn't have much luck with dynaconf.

So this seems reasonable alternative.